meson.build

```diff
@@ -59,6 +59,11 @@ py = py.find_installation(required: get_option('python'))
 swig = find_program('swig', required: get_option('python'))
 
 version_gen_h = vcs_tag(
+  command: [
+    'scripts/setlocalversion',
+    meson.current_source_dir(),
```

Comment:
If `vcs_tag` is already generating a version from the git information, what is it using `setlocalversion` for?

Reply:
So, on my local build vcs_tag() was not including the git info, it was just printing the fallback, which is the project version "1.7.0". I thought that was how vcs_tag() was supposed to work, and so we would need to inject it with setlocalversion.

No, turns out this is an open bug in Meson when using git-worktrees. I tried running this again in a non-worktree and it works as expected. So this Pull Request is not needed.
https://www.github.com/mesonbuild/meson/issues/8567

Reply:
Aha, thanks for the clarification.